### PR TITLE
Get info feature

### PR DESCRIPTION
**Pull request by gauridhawan**
# Pull Request Template

## Checklist:

- [x] I have written test cases for this change, and my code passed the test
- [x] I have updated documentation to reflect this change
- [x] I have attached this PR to an issue/user story

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -165,8 +190,7 @@ def create_shopcart_items():
     shopcart_item.deserialize(request.get_json())
     shopcart_item.create()
     message = shopcart_item.serialize()
-    #TODO: location_url = url_for("get_shopcart_items", 
-    #                              shopcart_item_id=shopcart_item.id, _external=True)
```
Comment (KHafner) [inline service/routes.py:169]: Can you leave this in? this code needs to be implemented as part of issue #36

Comment (gauridhawan) [reply service/routes.py]: Added!

**Comment by KHafner** [comment]
As part of this story don't we also need to add the ability to have a get request on /shopcarts/id and have that shopcart returned

File: service/routes.py [hunk 2]
```diff
@@ -149,6 +150,30 @@ def create_shopcarts():
...
+    This endpoint will return items in the shop cart 
+    """
+    app.logger.info("Request to get items in a shopcart")
+
+    shopcart_items =  ShopcartItem.find_by_shopcartid(shopcart_id)
+    
+
+    if shopcart_items is None or len(shopcart_items)==0:
+        app.logger.info("Shopcart with ID [%s] is empty.", shopcart_id)
+        return make_response(jsonify({}), status.HTTP_404_NOT_FOUND)
```
Comment (KHafner) [inline service/routes.py:169]: You could also use the error handler defined above here

**Comment by gauridhawan** [comment]
@KHafner in the story it says return the information of the items in the shopcart, but I can also return the shopcart details.

File: service/routes.py (hunk 1 above)
Comment (KHafner) [reply service/routes.py]: Thanks

**Comment by KHafner** [comment]
Yes if you can add it that would be great. In the future we should be more explicit when writing stories, but I think we have one story for each CRUD operation and they need to be implemented for both the shopcarts and shopcart items

**Comment by gauridhawan** [comment]
@KHafner got it, I'll add that!

**Comment by gauridhawan** [comment]
@KHafner added!

**Comment by KHafner** [comment]
Can you add the tests for find() in test_models.py as well

**Review by jules203 — APPROVED** [review]
Checks passed!

**Comment by gauridhawan** [comment]
@KHafner updated!

**Review by KHafner — APPROVED** [review]
Great, thanks!

**Pull request merged**